File: BetacomareFront/src/app/features/models/shoppingCart.ts
/* -- Carrello ordine effettuato -- */
export interface ShoppingCart{
    username:           string;   // username che ha effettuato l'ordine
    //orderId:            number;   // ID dell'ordine (numero d'ordine)
    productId:          number;   // ID del prodotto acquistato in quell'ordine
    qty:                number;   // quantita' del prodotto in questione
    //addressId:          number;   // indirizzo di spedizione
    name:               string;
    productNumber:      string;
    size:               string;
    color:              string;
    weight:             number;
    listPrice:          number;
}

File: Betacomare/ModelsBetacomare/ShoppingCart.cs
using System;
using System.Collections.Generic;

namespace Betacomare.ModelsBetacomare;

public partial class ShoppingCart
{
    public string Username { get; set; } = null!;

    public int OrderId { get; set; }

    public int ProductId { get; set; }

    public int Qty { get; set; }

    public int AddressId { get; set; }

    public virtual Address Address { get; set; } = null!;
}
